driverinference/main.py

```diff
@@ -1,9 +1,20 @@
 from utils import *
 from driverprocessresults.main import *
+import time
 
 if __name__ == "__main__":
+    """ Takes the pwd, then navigates to the root to append packages.
+    Python is then able to find our *.py files in that directory.
+    """
     sys.path.append("..")
 
+if __name__ == "__main__":
```

Comment:
okay so I asked matt drozt, he said that we will need sys.append() to run before the import utils, so I actually made a mistake with having it after and thank u for catching that!! He says "if you are executing directly via driver*/main.py instead of the driver.py script thus the multiple  if __name__ == '__main__': blocks. you could let sys.path.append(...) run unconditionally, but that will pollute the PYTHONPATH when executing driver.py directly. That said, it wont break anything so its not that big a deal"

should I combine the two still?

Reply:
I see. Sounds good then